gpio/zynq7000-gpio/gpiosrv.c

```diff
@@ -174,19 +174,14 @@ static void gpiosrv_msgthr(void *arg)
 					else {
 						err = gpio_writePin(bank, pin, val);
 					}
-
-					if (err < 0) {
-						msg.o.io.err = err;
-					}
-					else {
-						msg.o.io.err = msg.i.size;
-					}
+					msg.o.io.err = (err < 0) ? err : msg.i.size;
```

Comment:
Sneaky change, not related to the commit topic. But it's ok

Reply:
Didn't want to create separate commit to smuggle this change :P